### ISSUE TITLE
Can't build as frameworkless target (update instructions)

### ISSUE THREAD

**Issue by chopson** [stephencelis/SQLite.swift#407]
Using Xcode 7.3 w/ Swift 2.2 to build Mac target.
Just grabbed a copy of SQLite.swift.  I’m trying to use it in a command line tool.  I (carefully?) followed the steps in the docs, but I get many errors - all seem to cascade from the import CSQLite statement, i.e. No such module.  What am I missing?

FWIW, same target builds if I remove the SQLite source and link against the SQLite.framework.

Thanks for your help!

**Comment by t4ec** [comment]
Hi, same issue here: I've been using SQLite.swift for a while and there was no such issue before. I have just descided to update framework, grabbed fresh version from master branch and have the same problem: "No such module 'CSQLite'", I use frameworkless installation (iOS7)

**Comment by t4ec** [comment]
I have just rolled back to the older version I had before and have the same issue: seems like it is something to do with new version of Xcode

**Comment by mbilovus** [comment]
I've faced the same problem.
There is a similar problem described on the [stackoverflow](http://stackoverflow.com/questions/36700507/cant-build-command-line-project-with-sqlite-swift).
There is a comment that commenting out import CSQLite in three files Connection, Helpers, and Statement fixes the errors, but after doing this I got even more compiler errors.

I have two Xcode instances installed - one 6.2 in /Applications and another 7.3 in ~/Applications.
My project is iOS app on Swift with SQLite pod. The SQLite pod has a file at <project_folder>/Pods/SQLite.swift/CocoaPods/iphonesimulator/module.modulemap
The file had the content

> module CSQLite [system] {
>     header "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/usr/include/sqlite3.h"
>     export *
> }
, so it is referencing the sqlite3.h from the Xcode 6.2.
To fix this I did sudo xcode-select -switch ~/Applications/Xcode.app/ to select the active Xcode 7.3
Also I changed the content of module.modulemap to

```
module CSQLite [system] {
    header "/Users/<my_user_name>/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator.sdk/usr/include/sqlite3.h"
    export *
}

```
These steps made my project compiled successfully.
Hope this will be helpful.
